app/forms/password_reset_email_form.rb

```diff
@@ -0,0 +1,38 @@
+class PasswordResetEmailForm
+  include ActiveModel::Model
+  include FormEmailValidator
+
+  attr_accessor :email
+
+  def initialize(email)
+    self.email = email
```

Comment:
No particular reason. This was me not cleaning up after myself. I'll revert to attr_reader + ivar.